ky-universal/node.js

```diff
@@ -1,4 +1,4 @@
-const fetch = require('node-fetch')
+const fetch = require('node-fetch/lib/index.js')
```

Comment:
Should also be able to do:

```suggestion
const { default: fetch } = require('node-fetch')
```